public/stylesheet.css

```diff
@@ -1,11 +1,14 @@
 * {
+  border: 0;
```

Comment:
There's another `border: 0;` in the stylesheet, can you remove that one?

Reply:
Done, there was one more `border:none;` removed that too

Reply:
Missed that myself, thanks for checking and removing :ok_hand: